docs/api/authentication.md

```diff
@@ -1,4 +1,14 @@
 # Authentication
 
-- API Key (CoderSessionToken)
-  - Parameter Name: **Coder-Session-Token**, in: header.
+Long-lived tokens can be generated to perform actions on behalf of your user account:
+
+```console
+coder tokens create
+```
+
+You can use tokens with the Coder's REST API using the `Coder-Session-Token` HTTP header.
```

Comment:
Suggestion (minor): Reference what tokens and reduce usage of "use".

```suggestion
You can use the created tokens with Coder's REST API by setting the `Coder-Session-Token` HTTP header.
```

Reply:
Fixed!